src/components/Header/index.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState } from "react";
 import "./styles.scss";
-import headerBackground from "../../assets/images/bg_header.png";
+// import headerBackground from "../../assets/images/bg_header.png";
```

Comment:
`Instead` of commenting this line, please add a module mapper rule to provide a mock.

package.json:

  "jest": {
    "moduleNameMapper": {
      "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
      **"\\.(png)$": "<rootDir>/__mocks__/styleMock.js"**
    }
  },